### DISCUSSION TITLE
How to stream GoPro combined with a DSLR viewfinder to Twitch via Android out in nature

### DISCUSSION THREAD

**Discussion by SanderMuller** [gopro/labs#789] [General]
**Overview:**
Hello, GoPro Labs community! I'm facing a challenging task related to streaming as a nature/macro photographer. The GoPro support team directed me here, hoping to tap into your expertise.

**Setup:**
- GoPro: Hero 12
- Canon Camera: R5 (DSLR)
- Android Phone: Samsung S23

**Objective:**
I aim to stream both the video from my GoPro and the viewfinder/display of my Canon R5 to Twitch using mobile data. I plan to send the raw output of both sources to my Android phone and then use an app to stream to Twitch.

**Challenges:**
1. I am struggling to select my Canon R5's video output as a source in mobile streaming apps.
2. I need to understand how to utilize GoPro Labs to enable raw data streaming and combine it with the source of my Camera.
3. I'm looking for a solution compatible with outdoor settings (away from a PC).

**Specific Questions:**
- How can I select the camera sources in apps like Streamlabs/MBS Studio on my Android phone? Either directly or via an app like qDslrDashboard or even Canon Connect (but that only works over WiFi so would require more mobile data)
- Any insights on leveraging GoPro Labs for raw data streaming and combining sources?
- Any app recommendations for outdoor streaming?

I would be sincerely grateful for any insights, suggestions, or recommendations you can provide. Thank you in advance for your help! 🙏

**Comment by dnewman-gpsw** [comment]
Sorry, no idea.  Labs is custom firmware that extends some camera features, but for Live streaming not so much.  Labs add livestreaming without using Quik, to start to RTMP end point, that is about it. See https://gopro.github.io/labs/control/rtmp/

**Reply by SanderMuller** [reply]
Thank you for your quick answer. Hopefully someone has some experience in this direction and knows some chain of tooling to get this to work 😄
